harvester/harvester/supervisord/celery.conf

```diff
@@ -1,19 +0,0 @@
-[program:celery]
-command=celery -A harvester worker --concurrency=2 --loglevel=info --max-tasks-per-child=10 --pool=solo
```

Comment:
There is no container that uses this command. So there won't be any workers to pick up tasks. Or did I miss something?

Reply:
Oops. Forgot that one, pretty essential 😄